spacy_llm/models/hf/openllama.py

```diff
@@ -75,20 +77,22 @@ def compile_default_configs() -> Tuple[Dict[str, Any], Dict[str, Any]]:
         )
 
 
-@registry.llm_backends("spacy.OpenLLaMa_HF.v1")
-def backend_openllama_hf(
-    model: str,
+@registry.llm_models("spacy.OpenLLaMA.v1")
+def openllama_hf(
+    name: Literal[
+        "open_llama_3b_350bt_preview",
+        "open_llama_3b_600bt_preview",
+        "open_llama_7b_400bt_preview",
+        "open_llama_7b_600bt_preview",
+    ],  # noqa: F722
```

Comment:
Is there a way to avoid hard-coding these twice, both here and in `get_model_names`?

Reply:
A suggestion for that: https://github.com/explosion/spacy-llm/pull/176/commits/b068e1d1542e5a0c8f141469d1bb9a284ba144ce.